docs/concepts/artifacts.md

```diff
@@ -9,36 +9,43 @@ search:
 ---
 # Artifacts
 
-Artifacts are persisted outputs such as tables, markdown, or links. They can be published via the Prefect SDK or REST API. They are stored on Prefect Cloud or a Prefect server instance and rendered in the Prefect UI. Artifacts make it easy to track and monitor the objects that your flows produce and update over time.
+Artifacts are persisted outputs such as tables, Markdown, or links.
+Artifacts make it easy to track and monitor the objects that your flows produce and update over time.
+Artifacts  are stored on Prefect Cloud or a Prefect server instance and rendered in the Prefect UI.
```

Comment:
I think the flow of the paragraph breaks when all three sentences start with "artifact".

Reply:
Good call. Maybe best to put into a list. Ambiguous antecedents of pronouns make me hesitant to use them. I'll make an improvement of some sort here.

Reply:
👍